src/components/SingleArticle/SingleArticle.jsx

```diff
@@ -52,7 +58,22 @@ function SingleArticle() {
           setIsSubmitting(false);
         });
     }
-  };
+  }
+
+  function handleDelete(comment_id) {
+    const username = "happyamy2016";
+    deleteComment(comment_id, username)
+      .then(() => {
+        setComments((postedComments) =>
+          postedComments.filter((comment) => comment.comment_id !== comment_id)
+        );
+        setNotification("Comment deleted successfully");
+        setTimeout(() => setNotification(""), 3000);
+      })
+      .catch((error) => {
+        console.error("Failed to delete comment:", error);
```

Comment:
You could and should inform the user that they cannot delete a comment. The same way you have handled feedback for a successful deletion would be great.